package.json

```diff
@@ -106,8 +108,8 @@
     "kue": "^0.11.6",
     "lodash": "^4.0.0",
     "mocha": "^3.1.2",
-    "multer": "^1.0.3",
     "moment": "^2.18.1",
+    "multer": "^1.0.3",
```

Comment:
why?

Reply:
I think npm sorted packages alphabetically automatically

Reply:
oh yeah i didn't realize this was *moving* (I just thought you were adding it, and I was like "we are already handling multipart why are you adding this now?" but now I get it)